src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/EnergyNetComponent.java

```diff
@@ -66,6 +66,8 @@ default boolean isChargeable() {
      */
     default int getCharge(@Nonnull Location l) {
         Validate.notNull(l, "Location was null!");
+        if (!isChargeable()) return 0;
```

Comment:
eehhh, not exactly great but I get why it needs to be here for this.
Can you at least refactor it?

```suggestion

        // Emergency fallback, this cannot hold a charge, so we'll just return zero
        if (!isChargeable()) {
            return 0;
        }
```